guides/introduction/welcome.md

```diff
@@ -0,0 +1,143 @@
+# Welcome
+
+Welcome to the LiveView Native documentation. LiveView Native is a platform built on
+[Phoenix LiveView](https://github.com/phoenixframework/phoenix_live_view) designed for
+building native applications. A general overview of LiveView Native and its benefits
+[can be seen in our README](https://github.com/live_view_native/live_view_native).
+
+## What is LiveView Native?
+
+LiveView Native is an extention of Phoenix LiveView, designed to serve
+platform-specific server-rendered markup to a variety of supported clients.
+
+Since LiveView Native is built on Phoenix LiveView, LiveView Native inherits all
+the same rendering benefits of Phoenix LiveView while maintaining a familiar developer
+experience – making it a good pick for both your new and existing LiveView servers.
+
+To begin with LiveView Native, a basic understanding of Elixir and Phoenix LiveView
+is recommended. [You can find the documentation for Phoenix LiveView here](https://hexdocs.pm/phoenix_live_view/welcome.html).
+
+## How does LiveView Native work?
+
+To understand the fundementals of LiveView Native, it is important to analyze LiveView Native
+and its relationship to Phoenix LiveView and LiveView Native's rendering packages.
+
+### Querying
+
+Unlike Phoenix LiveView, when a client makes a request to a LiveView Native route, the route expects additional query
+parameters to be provided to denote platform-specific information from the client. This information is what allows
+LiveView Native to delegate the request to the appropriate rendering package and codepath, and
+maintain support with Phoenix LiveView.
+
+The query parameters are as follows.
+
+| Query Parameter | Arguments                | Required? | Description                                            |
+|-----------------|--------------------------|-----------|--------------------------------------------------------|
+| `_format`       | swiftui, jetpack, html   | ✅        | The content type to be processed by LiveView Native    |
+| `_interface`    | mobile, watch, tv        |           | The general device type                                |
+| `os_version`    | `string`                 |           | The version of the client OS                           |
+| `app_version`   | `string`                 |           | The version of the client application                  |
+
+This is formatted as `/?_format=xx&_interface=xx&os_version=xx&app_version=xx`, and when no query parameters are provided,
+will default to the corresponding Phoenix LiveView route (presuming a route is provided).
```

Comment:
Just FYI that we're looking at moving over to headers instead of query params. The QPs I think will continue to work as a fallback.

Reply:
That change won't take place until 0.5.0 if it does at all